students/slawinski_amadeusz/lesson_02_flow_control/4.05_lost_card.py

```diff
@@ -0,0 +1,13 @@
+# find missing card number in set of cards
+
+cards = int(input())
+
+sum_of_all = 0
+for c in range(1, cards + 1):
+    sum_of_all += c
+
+sum_of_remaining = 0
+for c in range(1, cards):
+    sum_of_remaining += int(input())
```

Comment:
można to z optymalizować i zamiast liczyć sumę wartości wprowadzanych kart, można je odejmować od oczekiwanej sumy i na końcu zostanie brakująca karta